### PR TITLE
Add io json

### PR DESCRIPTION
**Pull request by henrikstranneheim**
## Description

Use IO controller to read all JSON files.

### Added

- IO controller for JSON formatted files
- Add tests
- Add validation of path suffix for JSON files

### Changed

- Use controller to read JSON files

### Fixed

- Proper file suffix for Tower JSON files

### How to prepare for test
- [ ] ssh to hasta (depending on type of change)
- [ ] activate stage: `us`
- [ ] request trailblazer-stage on hasta: `paxa`
- [ ] install on stage:
    ```Shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_trailblazer -t trailblazer -b add_io_json -a
    ```
- [ ] ssh to clinical-db (depending on type of change)
- [ ] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-ui-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -132,7 +131,10 @@ def fixture_case_id() -> str:
     return CaseIDs.RUNNING
 
 
-@pytest.fixture(name="tower_task")
+@pytest.fixture(name="tower_task", scope="session")
```
Comment (seallard) [inline tests/conftest.py:134]: 👍 This fixture is probably pretty expensive to setup since it reads a file

**Review by seallard — APPROVED** [review]
Nice 👍 
I would remove the random name (cuddlyhen) from the file names, unless there is some reason to keep it. Maybe names like `tasks_completed.json` and `workflow_empty.json` is enough.

File: tests/conftest.py (hunk 1 above)
Comment (henrikstranneheim) [reply tests/conftest.py:134]: Yeah, I tried but pytest complains. Do not want to spend time on it now as we will remake the sql session handling in the future and the test suite is rather fast as it is.

**Comment by henrikstranneheim** [comment]
> Nice 👍 I would remove the random name (cuddlyhen) from the file names, unless there is some reason to keep it. Maybe names like `tasks_completed.json` and `workflow_empty.json` is enough.

👍 Fixed

**Pull request merged**